tools/test/windows/tw.cc

```diff
@@ -409,6 +409,14 @@ bool UnsetEnv(const wchar_t* name) {
   }
 }
 
+bool AddCurrentDirectoryToPATH() {
+  std::wstring path;
+  if (!GetEnv(L"PATH", &path) || !SetEnv(L"PATH", L".;" + path)) {
```

Comment:
From internal reviewer: 
this is probably more readable as simply "return GetEnv(...) && SetEnv(...);"